Базовая стилизация всех страниц

PolyTS opened the pull request:
---
:mortar_board: [Базовая стилизация всех страниц](https://up.htmlacademy.ru/htmlcss/35/user/2075923/tasks/8)

:boom: https://htmlacademy-htmlcss.github.io/2075923-device-35/4/

PolyTS committed:
Базовая стилизация всех страниц

PolyTS committed:
Revert "Базовая стилизация всех страниц"

This reverts commit 93184dbd01111281228dc4f0df5ac0a33990063a.

PolyTS committed:
"Базовая стилизация всех страниц"

PolyTS committed:
Update index.html

PolyTS committed:
Update catalog.html

pavel-niukalo commented on index.html:
```diff
@@ -210,25 +212,29 @@ <h2 class="services-title">Доставка</h2>
           Если из-за возгорания купленного у нас товара у вас сгорит дом — не
           переживайте, мы выдадим вам новый. Товар, не дом, конечно же.
         </p>
+        <!--вставила спан, просто не понимаю как без этого тогда вставить фоновую картинку?-->
```
Тут нам скорее всего потребуются обертки для заголовка и текста (чтобы можно было скрыть другие слайды как раз) и вот обертке можно дать фоновую картинку
```html
<div>
  <h2>Доставка</h2>
  <p>....</p>
</div>

<div>
  <h2>Гарантия</h2>
  <p>....</p>
</div>

<div>
  <h2>Кредит</h2>
  <p>....</p>
</div>
```

pavel-niukalo commented on styles/styles.css:
```diff
@@ -0,0 +1,400 @@
+  font-family: "Raleway";
+  font-style: normal;
+  font-weight: 800;
+  src: url(../fonts/raleway/raleway-800.woff2) format(woff2);
```
Для надежности лучше добавить кавычки:
`url("../fonts") format("woff2")`

А вот для `format` обязательны кавычки, ибо он так даже не подключает шрифты и ругается
![image](https://user-images.githubusercontent.com/52506235/197976249-749bb2da-0947-4e55-8117-19367367d492.png)

pavel-niukalo commented on styles/styles.css:
```diff
@@ -0,0 +1,400 @@
+  color:#000000;
+}
+/*Header*/
+/*Вот думаю этот блок наверное лучше не к хедеру присвоить, а сделать отдельный див и к нему этот цвет*/
```
Тут можно будет сделать через градиент с резким переходом, пока давай так оставим

pavel-niukalo commented on styles/styles.css:
```diff
@@ -0,0 +1,400 @@
+.page-header{
+  background-color: #FFE17F;
+}
+.navigation-list,
+.navigation-list-more{
+  list-style-type: none;
+}
+.navigation-search,
+.navigation-find{
+  font-family: inherit;
+  font-size: 16px;
+  line-height: 20px;
+  text-align: center;
+}
```
Чисто для удобства давай добавим отступов и пробелов
а) между селектором и `{`
б) между блоками селекторов

То есть вот так:
```css
.navigation-list,
.navigation-list-more {
  list-style-type: none;
}

.navigation-search,
.navigation-find {
  font-family: inherit;
  font-size: 16px;
  line-height: 20px;
  text-align: center;
}
```

pavel-niukalo commented on styles/styles.css:
```diff
@@ -0,0 +1,400 @@
+}
+/*Main*/
+/*эта цифра у меня белая, но сейчас она в блоке с белым фоном, полагаю это поправится с помощью сетки, как быть сейчас?
+и еще почему-то отличается шрифт, хотя это тоже rubik */
```
Шрифты вообще не подключились (выше указал почему) и сейчас на сайте везде `sans-serif`

![image](https://user-images.githubusercontent.com/52506235/197976516-d0b15e8e-48d3-4348-8af6-1857e90296fa.png)

pavel-niukalo commented on styles/styles.css:
```diff
@@ -0,0 +1,400 @@
+  height: 160px;
+}
+/*servises*/
+/*тут тот же вопрос насчет div, лучше через него цвета присваивать, а не всмему блоку?*/
```
Аналогично через градиент предлагаю сделать

pavel-niukalo commented on styles/styles.css:
```diff
@@ -0,0 +1,400 @@
+}
+/*добавила декоротивные картинки*/
+.delivery-man-icon{
+  background-image: url(../images/delivery-man-icon.svg);
```
Тут так же про кавычки

pavel-niukalo commented on styles/styles.css:
```diff
@@ -0,0 +1,400 @@
+    list-style-type: none;
+}
+.services-button{
+  font-family: 'Raleway',sans-serif;
```
Для единообразия надо использовать одинаковые кавычки по всему файлу, до этого например везде были двойные (строка 115 например)

pavel-niukalo commented on styles/styles.css:
```diff
@@ -0,0 +1,400 @@
+}
+/*тоже декоративное изображение*/
+.information-icon{
+  background-image: url(../images/information-icon.svg);
```
Про кавычки внутри `url` ко всему относится

PolyTS committed:
Дополнительные правки "Базовая стилизация всех страниц"

1. Убрала спаны, которые добавляла декоративным картинкам
2. Добавила кавычки, которых не хватало в стилях
3. Добавила отступы между селектором и между блоками селекторов
4. Привела к единообразию кавычки

pavel-niukalo commented on styles/styles.css:
```diff
@@ -0,0 +1,470 @@
+}
+
+.okay-icon {
+  background-image: url(../images/tick-big.svg);
```
Вот тут забыла кавычки
